plugins/@grouparoo/mailchimp/src/initializers/plugin.ts

```diff
@@ -34,8 +34,16 @@ export class Plugins extends Initializer {
         MailchimpAppTemplate,
         MailchimpSourceTemplate,
         MailchimpPropertyTemplate,
-        MailchimpEmailDestinationTemplate,
-        MailchimpIdDestinationTemplate,
+        new DestinationTemplate(
+          "mailchimp:email",
+          [path.join(templateRoot, "destination", "email", "*.template")],
+          ["sync", "additive", "enrich"]
```

Comment:
Just in case we change these later, you could export the supported sync modes from where you define the connections and use them here. See facebook for an example:

https://github.com/grouparoo/grouparoo/blob/facebook-sync-modes/plugins/%40grouparoo/facebook/src/initializers/plugin.ts#L7-L10

https://github.com/grouparoo/grouparoo/blob/facebook-sync-modes/plugins/%40grouparoo/facebook/src/initializers/plugin.ts#L32-L36